src/index.html

```diff
@@ -11,6 +11,26 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="https://www.google.com/" 
+      method="post" 
+      data-qa="big"
+    >
+      <input class="bar"
+        type="text" 
+        name="bigSearch" 
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
```

Comment:
```suggestion
      <input 
        class="bar"
        type="text" 
        name="bigSearch" 
        placeholder="Try &quot;Los Angeles&quot;"
      >
```
Each attribute should be placed on a new line